front/src/infoBox/InfoBox.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import Close from '@material-ui/icons/Close';
+import Alert from '@material-ui/lab/Alert';
+import api from '../api/api';
+import css from './InfoBox.module.scss';
+
+const classes = classNames.bind(css);
+
+// IMPORTANT: ONLY SUPER USER SHOULD BE ALLOWED ON THIS PAGE
```

Comment:
Infobox is used in dayview, weekview and monthview, all usually accessible to all users. Is this comment relevant?